### ISSUE TITLE
Markdown table output for the build/fuzzer test results?

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#214]
We have a very nice fuzzer, linter, build tester, etc. and it crossed my mind that we might want a way to format the results nicely for GitHub issues or file reports.  (Though we could always just copy paste images from the HTML interface).  But I was imagining an issue that shows the results of the build/fuzz for all sims so it is easy for everyone to see what needs to be fixed.

**Comment by zepumph** [comment]
I don't think this is needed, but if you still want this, please make an issue in aqua/
